### DISCUSSION TITLE
BUG

### DISCUSSION THREAD

**Discussion by augustoaraujoo** [community/community#32967] [Other Feature Feedback, Questions, & Ideas] [answered]
there was a bug on github, github was created in 2008 and I don't know how but there was a bug on the date of creation of this account, it says it was created in 1990 this is impossible lol

![image](https://user-images.githubusercontent.com/81258318/190020899-73114602-8fd2-4317-a9b0-320bfadd3bae.png)

**Comment by radar-blip** [comment, marked as answer]
Thats pretty funny 😆 
Anyway, if you want to file a bug report, you can do so here: [Bug-Report](https://support.github.com/contact/bug-report)

**Reply by augustoaraujoo** [reply]
Thanks man I was trying to find the link to report hehehehe

**Reply by radar-blip** [reply]
Not a problem. Glad I could help!
Have a nice flight!

**Comment by krunal882** [comment]
i found an bug in github , in profile section , when we change our profile picture then there is an hover effect , when we hover on emoji then outline is extending , after clicking one time , that outline is not extending

**Comment by AliHabibnia** [comment]
Hi GitHub team,
I’ve noticed that my Starstruck achievement shows only one repository and a multiplier of x2. However, I currently have multiple public repositories with at least 16 stars, none of which are forks or private. Can you please confirm whether this is a known issue or if my other repositories should be reflected in the multiplier?
